hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -57,8 +58,9 @@ def __init__(
         :param batch_size: Number of slides to load per batch.
         :param seed: pseudorandom number generator seed to use for shuffling instances and bags. Note that randomness in
         train/val/test splits is handled independently in `get_splits()`. (default: `None`)
-        :param transform: A transform to apply to the source tiles dataset, or a composition of
-        transforms using `monai.transforms.Compose`. By default (`None`).
+        :param transform: A dictionary that contains transform, or a composition of transforms using
+        `monai.transforms.Compose`, to apply to the source tiles dataset at training, validation and testing time.
```

Comment:
Do these just apply to tiles, or slides too?

Reply:
well spot, this is generic, fixed now